soc/rapidsilicon/virgo/soc.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2024 Rapid Silicon.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * VIRGO SoC specific interrupt map and mask functions
+ */
+
+#include <zephyr/kernel.h>
```

Comment:
Done. Though I think we could have included more information related to soc in it. Normally, I have seen that other vendors keep extra stuff in soc.c and soc.h.

Reply:
yeah, I would agree. My only point was that since this is specifically to the scu and any user in his application while calling SCU APIs will immediately be able to recognize that these are SCU APIs from the name. could be wrong here and my warrant a change back to soc in the future but lets leave it as SCU for now